vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_statictensor.py

```diff
@@ -0,0 +1,140 @@
+import torch
+from typing import List, Union, Tuple, Callable
+from vllm.model_executor.layers.quantization.compressed_tensors.cutlass_gemm import (
+    cutlass_gemm_dq)
+from vllm.model_executor.layers.quantization.compressed_tensors.schemes import (
+    CompressedTensorsScheme)
+from vllm.model_executor.utils import set_weight_attrs
+from torch.nn import Parameter
+from vllm._C import ops
+
+__all__ = ["CompressedTensorsW8A8StaticTensor"]
+
+
+class CompressedTensorsW8A8StaticTensor(CompressedTensorsScheme):
```

Comment:
To keep this schemes, I think the scope should be:
- support `compressed` only (so no `fake_quant`)
- support `symmetric` only (so no `zero_points`)

If we want to support other cases, we can add more schemes)

We should name it `W8A8StaticPerTensorSymmetric`

Reply:
We can separate out compressed vs fake-quant. 
The zero_points are only here because of the examples I got from sparseml. can remove

Reply:
Oh the compression_config has ZP's regardless of whether sym or not.

That makes sense. Not really anything we can do then